tests/sklearn/test_sklearn_model_export.py

```diff
@@ -629,6 +631,32 @@ def test_pyfunc_serve_and_score(sklearn_knn_model):
     np.testing.assert_array_almost_equal(scores, model.predict(inference_dataframe))
 
 
+def test_sklearn_compatible_with_mlflow_2_4_0(sklearn_knn_model):
+    model, inference_dataframe = sklearn_knn_model
+    model_predict = model.predict(inference_dataframe)
+
+    # save test model
+    model_data_path = "tests/sklearn/test_resources/test_model/model.pkl"
+    with open(model_data_path, "wb") as out:
+        pickle.dump(model, out, protocol=pickle.DEFAULT_PROTOCOL)
+
+    assert Version(mlflow.__version__) > Version("2.4.0")
```

Comment:
Do we need this line?

Reply:
I think so, as the model is logged with mlflow 2.4.0 and the point of this test is to make sure mlflow with newer version is compatible with that